### PR TITLE
[IA-3092] Updating button format for 'Delete Environment'

### PR DESCRIPTION
**Pull request by xuscrbw**
<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Thanks!
--!>

https://broadworkbench.atlassian.net/browse/IA-3092

For the Compute Modal, I updated the wording from 'Delete Environment Option' to 'Delete Environment'. Also changed the button type to be ButtonOutline from ButtonSecondary to give it the look desired. (Screenshots are from the browser locally)
Screen Shot 2021-12-07 at 12 39 06 PM
Screen Shot 2021-12-07 at 12 33 50 PM
Screen Shot 2021-12-07 at 12 36 28 PM

Also updated the Galaxy Modal 'Delete' to look like 'Delete Environment'. I used the same ButtonOutline change 

I am a bit unsure about the look of the Button type when it is disabled. Would like some input on this.

Screen Shot 2021-12-07 at 12 33 59 PM
Screen Shot 2021-12-07 at 12 34 20 PM
Screen Shot 2021-12-07 at 12 38 43 PM

### CODE REVIEW

**Review by Qi77Qi — APPROVED** [review]
lgtm....tried locally and the button looks correct

**Comment by cahrens** [comment]
Is this "Delete Environment" button supposed to be disabled when it is in this state? For me it is enabled, and it does not have sufficient color contrast. Interestingly though, it has a tooltip stating that Cloud Compute must be resumed first.

image

File: src/components/ComputeModal.js [hunk 1]
```diff
@@ -396,7 +396,7 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
     return {
       hasGpu: computeConfig.hasGpu,
       runtime: Utils.cond(
-        [(viewMode !== 'deleteEnvironmentOptions'), () => {
+        [(viewMode !== 'deleteEnvironment'), () => {
```
Comment (dmohs) [inline src/components/ComputeModal.js:399]: Appreciate the attention to detail in renaming this.

**Review by dmohs — APPROVED** [review]
Approving for code quality. Please address concern raised by @cahrens before merge.

**Comment by cahrens** [comment]
> Is this "Delete Environment" button supposed to be disabled when it is in this state? For me it is enabled, and it does not have sufficient color contrast. Interestingly though, it has a tooltip stating that Cloud Compute must be resumed first.

It appears that the bug is in `ButtonOutline`. The disabled prop needs to be included in the `_.merge`, similar to `ButtonSecondary` right above it.

**Comment by xuscrbw** [comment]
Made the proposed change!

**Comment by cahrens** [comment]
Thanks @xuscrbw!

**Pull request merged**